torch/_inductor/compile_fx.py

```diff
@@ -681,6 +702,14 @@ def compile_fx(
 
     graph_id = next(_graph_counter)
 
+    orig_model = model_
+    if config.keep_output_stride:
+        # we need know the number of outputs of the original model.
+        # should do a make_fx if orig_model is not a GraphModule yet.
+        # This happens if people call inductor without dynamo.
```

Comment:
When are people calling inductor without dynamo? I would probably just default to keeping all the output strides the same in that case.

Reply:
I encountered some CI failures and this is to fix them. Those usages I see are in: https://github.com/pytorch/pytorch/blob/main/test/inductor/test_standalone_compile.py .

We can disable layout optimization in that case, but I think it can be beneficial to enable it for potential perf wins in those scenarios?

Reply:
But I can do what you said if we don't care much about using inductor not thru dynamo. That can simplify code a bit.